### PR TITLE
docs: add information about multiple sources

### LINKED ISSUE #130: Use multiple sources within the same project
**Before you submit:**

- [x] Please search through the existing issues (both open AND closed) to see if your question has already been discussed. Github issue search can be used for this: https://github.com/imgix/gatsby/issues?utf8=%E2%9C%93&q=is%3Aissue

**Question**
Is it possible to use images from multiple imgix sources within the same Gatsby project? E.g. images from a CMS (through a `Web Proxy Source`) and images from a S3 bucket (through a `Amazon S3 Source`)?

Thanks for any hints.

### PR DESCRIPTION
**Pull request by frederickfogerty**
Also took the time here to do a little reorganising

Closes #130

### CODE REVIEW

**Review by sherwinski — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1143,6 +1147,51 @@ Gatsby does this in an effort to reduce the likelihood installing one plugin can
 
 You can read more about this if you’re interested in [this issue](https://github.com/imgix/gatsby/issues/129).
 
+### Multiple imgix Sources
+
+You might find yourself with multiple imgix sources and wondering how you could use them at the same time with this plugin. There are a few possibilities for this, which will be outlined below.
+
+Any number of sources can be used simultaneously with the URL transform API. The caveats with this approach is that you don't get the benefits of the GraphQL APIs (blur-up, etc), and that the sources must not be Web Proxy sources (for these you must use one of the GraphQL APIs).
```
Comment (luqven) [inline README.md:1154]: The only nit is that maybe it's better to add the `Only caveat` section after the example. This way I can _see_ what you mean a little sooner. But that's a personal preference, so feel free to ignore it.

**Review by luqven — APPROVED** [review]
LGTM 👍🏼

File: README.md (hunk 1 above)
Comment (frederickfogerty) [reply README.md:1154]: I can get behind this

**Review by luqven — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**